consensus/src/block_storage/sync_manager.rs

```diff
@@ -171,7 +171,11 @@ impl BlockStore {
         )
         .await?
         .take();
-        debug!("{}Sync to{} {}", Fg(Blue), Fg(Reset), root.0);
+        debug!(
+            LogSchema::new(LogEvent::CommitViaSync).round(self.root().round()),
+            committed_round = root.0.round(),
+            block_id = root.0.id(),
```

Comment:
It looks like you have `committed_round` and `block_id` in a couple different log lines, maybe it would be useful to push these up into the LogSchema type?